### PR TITLE
Allows the json-mappers.properties be found when running inside Eclipse IDE

### PR DESCRIPTION
**Pull request by cvgaviao**
Motivation:

The way the vertx-codegen annotation processor was searching for the json-mappers.properties doesn't work when running inside Eclipse IDE (Linux - java11).
Eclipse IDE has a limitation where a `processingEnv.getFiler().getResource()` don't return a file as expected.
These changes ensure that both maven and ide could properly find an existent properties file.
Also, it was added proper treatment of NullPointerException and some info/error messages.

### CODE REVIEW

**Comment by vietj** [comment]
@cvgaviao how can we test this change ?

**Comment by vietj** [comment]
how is this Pr different from https://github.com/vert-x3/vertx-codegen/pull/296/files ?

**Comment by cvgaviao** [comment]
> how is this Pr different from https://github.com/vert-x3/vertx-codegen/pull/296/files ?

Well, I was trying to understand exactly the reason github mixed the commits from the two PR. I think, but have no sure, it was because the first PR I haven't created a specific branch.

**Comment by cvgaviao** [comment]
> how is this Pr different from https://github.com/vert-x3/vertx-codegen/pull/296/files ?

Well, I was not able to look in detail how annotation processor testing is supposed to work. I just focused on not breaking any of those already there...

**Comment by cvgaviao** [comment]
> @cvgaviao how can we test this change ?

Well, I was not able to look in detail how annotation processor testing is supposed to work. I just focused on not breaking any of those already there...

File: src/main/java/io/vertx/codegen/CodeGen.java [hunk 1]
```diff
@@ -83,9 +83,11 @@ public void init(RoundEnvironment round, ClassLoader loader) {
       TypeMirror type = null;
       if (converter.className != null) {
         TypeElement typeElt = elementUtils.getTypeElement(converter.className);
-        if (typeElt != null) {
-          type = typeElt.asType();
-        }
+		if (typeElt != null) {
```
Comment (cvgaviao) [inline src/main/java/io/vertx/codegen/CodeGen.java:86]: @vietj, this seems to be easier to test. We just need to write a class in the json-mapper.property wrongly.

**Comment by cvgaviao** [comment]
I'm closing this PR in order to separate its commits from those from the PR 295

**Pull request closed without merging**